### PR TITLE
ansible-collections/news-for-maintainers#9 fixes

### PR DESCRIPTION
**Pull request by D3DeFi**
##### SUMMARY
Sanity tests are failing as advertised in ansible-collections/news-for-maintainers#9 

##### ISSUE TYPE
- Bugfix Pull Request

### CODE REVIEW

**Comment by D3DeFi** [comment]
@felixfontein related to https://github.com/ansible-collections/news-for-maintainers/issues/9, can you please provide some guidance here? Not sure how is this supposed to work with connection plugins

File: plugins/connection/httpapi.py [hunk 1]
```diff
@@ -6,8 +6,8 @@
 __metaclass__ = type
 
 DOCUMENTATION = """
-author: fischbacher.markus@gmail.com
-connection: zabbix
+name: zabbix
```
Comment (felixfontein) [inline plugins/connection/httpapi.py:9]:
```suggestion
name: httpapi
```
The plugin is named `httpapi`, not `zabbix`. See for example `ansible-doc -t connection -l community.zabbix`.

File: plugins/httpapi/jsonrpc.py [hunk 2]
```diff
@@ -9,8 +9,8 @@
 
 DOCUMENTATION = """
 ---
-author: fischbacher.markus@gmail.com
-httpapi : zabbix
+name: zabbix
```
Comment (felixfontein) [inline plugins/httpapi/jsonrpc.py:12]:
```suggestion
name: jsonrpc
```
Same here: the plugin's name is `jsonrpc`, not `zabbix`.

**Comment by D3DeFi** [comment]
thanks a lot @felixfontein ! :) I wouldn't guess

**Pull request merged**